CONFIGDOC.md

```diff
@@ -1,18 +1,20 @@
 You can use the Jenkins&trade; plugin for MATLAB&reg; in freestyle and multi-configuration projects. This guide demonstrates how to run MATLAB code on Jenkins, considering a Microsoft&reg; Windows&reg; platform.
 
--  [Add MATLAB to System Path](#add-matlab-to-system-path)
+-  [Use MATLAB for Build](#use-matlab-for-build)
 -  [Specify Build Steps](#specify-build-steps)
       - [Run MATLAB Tests](#run-matlab-tests)
       - [Run MATLAB Command](#run-matlab-command)
 -  [Set Up Freestyle Project](#set-up-freestyle-project)
 -  [Set Up Multi-Configuration Project](#set-up-multi-configuration-project)
 
-## Add MATLAB to System Path
+## Use MATLAB for Build
 Once you install the plugin on Jenkins, the **Use MATLAB version** checkbox appears in the **Build Environment** section of the project configuration window.
 
 ![matlab_path](https://user-images.githubusercontent.com/48831250/76883191-1c528b00-6852-11ea-9242-7d143f8a5c46.png)
 
-Select the **Use MATLAB version** checkbox to specify the full path to the MATLAB root folder that should be used for the build. The specified directory in the **MATLAB root** box is prepended to the PATH environment variable, and the matlab.exe file in the bin subdirectory is used to perform the build. You must not leave the **MATLAB root** box blank if no MATLAB directory exists on the path. If you do not select the checkbox or if you leave the text box blank, then the system tries to use the MATLAB directory closest to the top of the system path.
+Select the **Use MATLAB version** checkbox to specify the full path to the MATLAB root folder that should be used for the build. The information in the **MATLAB root** box enables the plugin to prepend MATLAB to the PATH environment variable of the build agent. 
+
+If the build agent already has your desired MATLAB on the path, then you are not required to select the **Use MATLAB version** checkbox. When the checkbox is clear, the plugin uses the MATLAB directory closest to the top of the path. The build fails if the plugin cannot find MATLAB on the path.

```

Comment:
This is great. Two small tweaks:

_When the checkbox is clear, the plugin uses the first MATLAB it encounters on the PATH.

 The build fails if MATLAB is not found on the path._

Doesn't have to be phrased  these ways, but for the first one "closest to the top of the path" just sounded a bit off. and the second sentence I just want to clarify because the plugin doesn't really find MATLAB. It just calls, MATLAB, and if the shell doesn't find it it will fail.

Reply:
Thanks, Andy. I tweaked the the sentences. If the text has any issues, please let me know.